Backpack/Font/Classes/Generated/BPKFont.m

```diff
@@ -38,55 +54,301 @@ + (UIFont * _Nullable)fontWithStyle:(BPKFontStyle)style {
+            case BPKFontStyleTextXxl:
+                return [UIFont systemFontOfSize:36 weight:UIFontWeightRegular];
+            case BPKFontStyleTextXxlEmphasized:
+                return [UIFont systemFontOfSize:36 weight:UIFontWeightSemibold];
+            case BPKFontStyleTextXxlHeavy:
+                return [UIFont systemFontOfSize:36 weight:UIFontWeightHeavy];
+            case BPKFontStyleTextXxxl:
+                return [UIFont systemFontOfSize:42 weight:UIFontWeightRegular];
+            case BPKFontStyleTextXxxlEmphasized:
+                return [UIFont systemFontOfSize:42 weight:UIFontWeightSemibold];
+            case BPKFontStyleTextXxxlHeavy:
+                return [UIFont systemFontOfSize:42 weight:UIFontWeightHeavy];
+            default:
+              NSAssert(NO, @"Unknown fontStyle %ld", (unsigned long)style);
+    }
+}
+
++ (NSNumber *_Nullable)trackingForStyle:(BPKFontStyle)style {
+    switch (style) {
+        
+            case BPKFontStyleTextBase:
+              
+                return @(0.12);
+              
+            case BPKFontStyleTextBaseEmphasized:
+              
+                return @(0.12);
+              
+            case BPKFontStyleTextCaps:
+              
+                return @(0.28);
+              
+            case BPKFontStyleTextCapsEmphasized:
+              
+                return @(0.28);
+              
+            case BPKFontStyleTextLg:
+              
+                return  nil;
+              
+            case BPKFontStyleTextLgEmphasized:
+              
+                return  nil;
+              
+            case BPKFontStyleTextSm:
+              
+                return @(0.154);
+              
+            case BPKFontStyleTextSmEmphasized:
+              
+                return @(0.154);
+              
+            case BPKFontStyleTextXl:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXlEmphasized:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXlHeavy:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXs:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXsEmphasized:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXxl:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXxlEmphasized:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXxlHeavy:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXxxl:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXxxlEmphasized:
+              
+                return  nil;
+              
+            case BPKFontStyleTextXxxlHeavy:
+              
```

Comment:
I'm all for letting the code breathe, but I think it might be breathing too much in this `switch` statement. Is there a reason it's so spaced out?

Reply:
Generated code and nunjucks, not sure how to make it tighter

Reply:
Fair enough - it's not a big deal